application/forms/Command/Object/AddCommentForm.php

```diff
@@ -21,6 +23,23 @@ class AddCommentForm extends CommandForm
 {
     use Auth;
 
+    public function __construct()
+    {
+        $this->on(self::ON_SUCCESS, function () {
+            $countObjects = count($this->getObjects());
+            $objectsType = 'services';
+            if (current($this->getObjects()) instanceof Host) {
+                $objectsType = 'hosts';
+            }
+
+            Notification::success(sprintf(
+                tp('Added comment successfully', 'Added comments to %d %s successfully', $countObjects),
```

Comment:
`comment` for the plural version. It's the same comment. Yes, multiple versions of it, but that's already stated by using the plural for the target.